lookout/style/format/features.py

```diff
@@ -274,7 +274,7 @@ def extract_features(self, files: Iterable[File], lines: List[List[int]]=None
                            (vnode.start.line in file_lines if file_lines is not None else True)])
 
         y = numpy.concatenate(labels)
-        X = numpy.full((y.shape[0], self.count_features(FeatureType.all)), -1)
+        X = numpy.full((y.shape[0], self.count_features(FeatureType.all)), -1, dtype=numpy.int8)
```

Comment:
But position features can overflow `int8`.

Reply:
Why not uint8 and 256?

Reply:
we have -1 to denote the absence of features

Reply:
Make it `0xff` then. MANY lines will exceed 127